### PR TITLE
Features/status change at top of supp app #157289076

### PR DESCRIPTION
**Pull request by chrisdolendo**
Only thing to figure out is why scrolling to an anchor reloads the page first.

### CODE REVIEW

**Comment by callaghanc** [comment]
@chrisdolendo From the mockup it looks like the status update component is meant to be situated in the supplemental application in a grey band that spans the width of the application: https://projects.invisionapp.com/share/ACEZNG43Y#/screens/269807121_MVP_5.

File: app/javascript/components/organisms/StatusUpdate.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import StatusDropdown from '../molecules/StatusDropdown'
 
 const StatusUpdate = ({ status, comment, date, onStatusDropdownChange, onAddCommentClick, statusHistoryAnchor }) => {
   return (
-    <div className='status-update'>
+    <div className={`status-update expand-wide is-${status.toLowerCase()}`}>
```
Comment (chrisdolendo) [inline app/javascript/components/organisms/StatusUpdate.js]: Waiting on @slowbot on whether to add `is-{status}` on the whole component div as part of the requirements, or just the gray (based on slack convo), if needed, can keep, if not needed, just remove that one class.

**Comment by chrisdolendo** [comment]
**Hand Off Todo's**:
1. Figure out is why clicking on "See Status History" link on component reloads the page before scrolling down to the bottom at the Status History section. to an anchor reloads the page first.
1. Confirm with @slowbot if `is-{status}` class should be part of the component. It is built as such currently, easy to remove if not needed.

**Comment by chrisdolendo** [comment]
@slowbot says yes on the ‘is-status’ class so let’s keep it 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: app/javascript/components/organisms/StatusUpdate.js [hunk 2]
```diff
@@ -33,7 +34,7 @@ const StatusUpdate = ({ status, comment, date, onStatusDropdownChange, onAddComm
           </button>
           {
             (status || comment) &&
-            <a href={statusHistoryAnchor} className='t-small right'>See Status History</a>
+            <a href={statusHistoryAnchor} data-turbolinks='false' className='t-small right'>See Status History</a>
```
Comment (callaghanc) [inline app/javascript/components/organisms/StatusUpdate.js:37]: @chrisdolendo Turbolinks was the culprit! It's apparently a known issue with Turbolinks that it causes page reload on anchor link clicks: https://github.com/turbolinks/turbolinks/issues/75. Disabling Turbolinks on the status history anchor link solves the page reload problem and allows the scroll-to-anchor behavior to proceed normally.

File: app/javascript/components/supplemental_application/sections/StatusList.js [hunk 3]
```diff
@@ -25,7 +14,7 @@ const getStatusClassName = (status) => {
   }
 }
 
-const StatusListItem = ({status, note, date}) => {
+const StatusListItem = ({status, comment, date}) => {
```
Comment (callaghanc) [inline app/javascript/components/supplemental_application/sections/StatusList.js:17]: All this is correcting our terminology that we use to refer to a Field Update Comment's comment from "note" to "comment" to maintain consistency.

File: app/javascript/components/mappers/soqlToDomain/field_update_comment.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import moment from 'moment'
 export const mapFieldUpdateComment = (item) => {
   return {
     status: item.Processing_Status,
-    note: item.Processing_Comment,
+    comment: item.Processing_Comment,
```
Comment (callaghanc) [inline app/javascript/components/mappers/soqlToDomain/field_update_comment.js:6]: This is correcting our terminology that we use to refer to a Field Update Comment's comment from "note" to "comment" to maintain consistency.

**Review by callaghanc — APPROVED** [review]

**Review by akegan — APPROVED** [review]

**Pull request merged**